### DISCUSSION TITLE
USB HID, at least USB mouse support

### DISCUSSION THREAD

**Discussion by jbrunk1966** [markadev/AppleII-VGA#48] [Ideas]
it would be great if we could attach a standard USB Mouse to the Pico (eg. via an USB2GO cable) and use it with eg. Apple Desktop or similar 👍

**Comment by markadev** [comment]
You can also check out the [a2pico](https://github.com/oliverschmidt/a2pico) project which is like a generic card that uses a Pico to emulate various hardware devices.
